test/Streamly/Test/Data/Array/Unboxed.hs

```diff
@@ -18,10 +21,16 @@ import Streamly.Internal.Data.Unboxed (Unboxed)
 import Test.QuickCheck (chooseInt, listOf)
 import GHC.Ptr (plusPtr)
 
+import qualified Streamly.Data.Fold as Fold
 import qualified Streamly.Internal.Data.Array.Unboxed as A
 import qualified Streamly.Internal.Data.Array.Unboxed.Type as A
```

Comment:
Do we need this? Array.Unboxed should have all we need.

Reply:
Functions from both modules are used.